src/nrncvode/netcvode.cpp

```diff
@@ -2918,6 +2917,27 @@ void NetCvode::clear_events() {
     }
 }
 
+// Frees the allocated memory for the SelfEvent pool and TQItemPool after cleaning them
+void NetCvode::free_event_pools() {
+    clear_events();
+    for (int i = 0; i < nrn_nthread; ++i) {
+        NetCvodeThreadData& d = p[i];
+        if (d.sepool_) {
+            delete d.sepool_;
+        }
+        if (d.selfqueue_) {
+            delete d.selfqueue_;
```

Comment:
Just wondering why this line doesn't get in the coverage: https://app.codecov.io/github/neuronsimulator/nrn/blob/magkanar%2Frefactor_eventqueues/src/nrncvode/netcvode.cpp

Reply:
I think this is because `d.selfqueue_` is only initialized if `nrn_use_selfqueue_` is set and this is only enabled in https://github.com/neuronsimulator/nrn/blob/master/test/pynrn/test_netpar.py#L97